packages/api/src/controllers/access-control.ts

```diff
@@ -270,37 +268,4 @@ app.get("/public-key", async (req, res) => {
     });
 });
 
-async function freeTierLimitReached(
```

Comment:
So happy to get rid of this! There's even a bonus: catalyst is also much more capable of actually limiting the number of _concurrent viewers_ instead of "number of viewers that start per minute"! Might be a lower hanging fruit for the future

Reply:
@gioelecerati do you think it is possible to limit the number of concurrent viewers for this work as Victor mentioned that it is a low hanging fruit and not worry about it in the future?

Reply:
Yes, just rebased it and tested, waiting for green and merge